### PR TITLE
typo fix.

### PR DESCRIPTION
**Pull request by fujitatomoya**
Signed-off-by: Tomoya Fujita <Tomoya.Fujita@sony.com>

### CODE REVIEW

File: rclcpp/src/rclcpp/executor.cpp [hunk 1]
```diff
@@ -913,7 +913,7 @@ Executor::get_next_executable(AnyExecutable & any_executable, std::chrono::nanos
   return success;
 }
 
-// Returns true iff the weak_groups_to_nodes map has node_ptr as the value in any of its entry.
+// Returns true if the weak_groups_to_nodes map has node_ptr as the value in any of its entry.
```
Comment (fujitatomoya) [inline rclcpp/src/rclcpp/executor.cpp:916]: Do we need a `s` in `Returns` ? since i am not native speaker, not sure... 😅

**Comment by alsora** [comment]
Is this really a typo?
`iff` could be a short hand for `if and only if`, however I don't think we need this kind of language in doxygen documentation.

**Comment by fujitatomoya** [comment]
> iff could be a short hand for if and only if

thanks! that is something i did not know 😅 if this is not typo, we can just go ahead to close this one!

**Comment by clalancette** [comment]
> > 
> 
> thanks! that is something i did not know sweat_smile if this is not typo, we can just go ahead to close this one!

I agree with Alberto in that this is likely "iff", meaning "if and only if".  That said, I think it would probably aid understanding (and avoid this ambiguity) by replacing it with "if and only if" where it is used.  @fujitatomoya If you go ahead and do that, I'm happy to approve.

**Comment by fujitatomoya** [comment]
i do not think that is necessary to fix, thanks for checking!

**Pull request closed without merging**